data-raw/test_data.R

```diff
@@ -36,6 +36,17 @@ forecast_horizon <- 28
 generation_interval <- wwinference::generation_interval
 inf_to_hosp <- wwinference::inf_to_hosp
 
+dist_matrix <- as.matrix(
```

Comment:
Related to my comment above and for a separate issue, I think you would want to just pull this in as package data corresponding to the spatially generated wastewater data and hospital admissions data, that way if you change it in the generate simulated data function and generate new package data it gets changed here too.